dom/nodes/insertion-effects/Node-appendChild-script-and-iframe-from-fragment.tentative.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Node.appendChild: inserting script and iframe from a div</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id="log"></div>
+<script>
+let happened = [];
+const iframe = document.createElement("iframe");
+test(() => {
+  iframe.src = "data:text/html,"
+
+  const script = document.createElement("script");
+  script.textContent = `
+    happened.push(iframe.contentWindow === null);
+`;
+
+  const df = document.createDocumentFragment();
+  df.appendChild(script);
+  df.appendChild(iframe);
+
+  assert_array_equals(happened, []);
+  document.body.appendChild(df);
+  assert_array_equals(happened, [false])
```

Comment:
```suggestion
  assert_array_equals(happened, [false], "Script inserted before iframe can observe iframe's contentWindow");
```

The point of this comment is to request an assertion description. Whether that is in the form of a positive description like I added or a negative one like you've added in some above, I don't feel strongly on.